modules/ipi-install-bmc-addressing.adoc

```diff
@@ -64,3 +64,63 @@ platform:
           password: <password>
           disableCertificateVerification: True
 ----
+[discrete]
+== Redfish APIs
+
+When running your bare-metal installer-provisioned infrastructure, you need to call several redfish API endpoints on your BMC.
```

Comment:
@kelbrown20 
Lets change this to

`Several redfish API endpoints on your BMC are called when the bare-metal installer-provisioned infrastructure is used.`

Reply:
How does this update look Jeff?